### DISCUSSION TITLE
post/120124-powerautomate-check-if-an-item-is-new-or-edited/

### DISCUSSION THREAD

**Discussion by giscus** [AlexanderHenkel/blogComments#8] [Announcements]
# post/120124-powerautomate-check-if-an-item-is-new-or-edited/

Learn how to see if an item has been created or modified, and create different automations based on that value

https://alexanderhenkel.dk/post/120124-powerautomate-check-if-an-item-is-new-or-edited/

**Comment by boby5555** [comment]
If the SinceVersionExisted is false when the item is created, it means your condition will return true. So I think the two send email actions should be inverted? Thanks for the article btw I helped me for a client !! :)

**Reply by AlexanderHenkel** [reply]
I am glad I was able to help you, solve your challenge with the client :)

You are right about the e-mail image! I have updated it c',)
